src/Logging/TeamCity.php

```diff
@@ -71,7 +71,7 @@ public function printResult(TestResult $result): void
     /** @phpstan-ignore-next-line */
     public function startTestSuite(TestSuite $suite): void
     {
-        if (!str_contains($suite->getName(), '.php')) {
+        if (str_starts_with($suite->getName(), 'P\\')) {
```

Comment:
str_starts_with is PHP 8 only?

Reply:
Lol. My bad. Will update.

Reply:
@lukeraymonddowning would it be possible to add a test that runs the teamcity logger? :) 

It's kinda bad that our CI did not catch this.

Reply:
This actually will succeed on <8.0 because its using a polyfill from symfony. Don't like that we're relying on that though so will alter anyway.